lzero/model/unizero_world_models/world_model.py

```diff
@@ -360,7 +360,8 @@ def precompute_pos_emb_diff_kv(self):
         if self.context_length <= 2:
             # If context length is 2 or less, no context is present
             return
-
+        if self.config.rotary_emb:
```

Comment:
remove these two lines since if is not necessary

Reply:
polished